### ISSUE TITLE
Add visually hidden text to provide additional context about reporting status

### ISSUE THREAD

**Issue by LucyGwilliamAdmin** [open-sdg/open-sdg#832]
**Issue from DAC report:**
The statuses of the indicators are visually formatted to look like badges which allows sighted users to infer the purpose of the information however, there is no alternative to this visual information.

**Solution from DAC report:**
Please consider adding some visually hidden text to provide the additional context which is provided visually.

Page 60 on DAC report

**Example from DAC report:**

```
<span class="visuallyhidden"> Indicator status: </span>
<span class="status complete">Reported online</span>
```

This would need to implemented on all goal page layouts

**Comment by LucyGwilliamAdmin** [comment]
Fix for this has been merged so I am closing
